packages/react/src/components/ComboBox/ComboBox.js

```diff
@@ -310,9 +312,11 @@ export default class ComboBox extends React.Component {
               })}>
               <input
                 className={`${prefix}--text-input`}
-                aria-label={ariaLabel}
+                aria-labelledby="combobox-id-a11y"
```

Comment:
We should make sure this is computed otherwise if there are two combo boxes on a page they have the same id being generated

Reply:
That makes sense! I added something to do this, let me know if it doesn't look right!